src/system.c

```diff
@@ -123,7 +123,26 @@ void sys_set_seccomp_syscall(bool enable)
  */
 int sys_chk_seccomp_action(uint32_t action)
 {
-	if (action == SCMP_ACT_KILL) {
+	int rc, nr_seccomp;
+	uint32_t kaction = action & SECCOMP_RET_ACTION_FULL;
```

Comment:
We need to prepare ourselves for building on system that doesn't define `SECCOMP_RET_ACTION_FULL`; it didn't exist before Kees' patch in August 2017.

Reply:
Nevermind, I see you define it in the patch below (src/system.h).

Reply:
Actually no, scratch that ... we could still be on a non cutting edge system which has seccomp.h but doesn't have SECCOMP_RET_ACTION_FULL defined; see how we handle SECCOMP_RET_LOG and friends in src/system.h.